topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin_jetbrains_runtime.md

```diff
@@ -0,0 +1,126 @@
+<!-- Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license. -->
+
+# JetBrains Runtime (JBR)
+
+<link-summary>The IntelliJ Platform Gradle Plugin requires JetBrains Runtime (JBR), which can be provided in multiple ways.</link-summary>
+
+<include from="tools_intellij_platform_gradle_plugin.md" element-id="Beta_Status"/>
+<include from="tools_intellij_platform_gradle_plugin.md" element-id="faq"/>
+
+The IntelliJ Platform Gradle Plugin requires JetBrains Runtime (JBR) for running IDE locally, testing, or performing other actions that require Java Runtime adjusted by JetBrains.
```

Comment:
what does "adjusted by JB" mean

Reply:
I would start with a sentence or two of explanation what JBR is and why it exists. Then that IPGP needs it too.